src/node_modules.cc

```diff
@@ -308,26 +297,22 @@ const BindingData::PackageConfig* BindingData::TraverseParent(
                  !env->permission()->is_granted(
                      env,
                      permission::PermissionScope::kFileSystemRead,
-                     std::string(check_path) + kPathSeparator))) {
+                     current_path.string()))) {
```

Comment:
Does this still need the trailing separator?

Reply:
I'm not sure. @tniessen @RafaelGSS